src/routes/chat/components/message/Message.js

```diff
@@ -1,17 +1,55 @@
-import React, { memo } from 'react';
+import React, { memo, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
 import useStyles from 'isomorphic-style-loader/useStyles';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-// After creating user model will use Avatar component instead <img>
-// import Avatar from '../avatar';
+import isEmpty from 'lodash/isEmpty';
+
+import Avatar from '../avatar';
 import IconMessageStatus from '../messageStatus/IconMessageStatus';
+import { setCurrentMessage } from '../../../../actions/security-chat';
+import deleteicon from './assets/deleticon.svg';
 
 import s from './Message.css';
 
 const testIsRead = true;
 
-function Message({ isMe, createdAt, text, user }) {
+function Message({
+  chatRemoveMessage,
+  changeInputToEdit,
+  currentUserId,
+  createdAt,
+  item,
+  messageId,
+  text,
+  user,
+}) {
   useStyles(s);
+  const dispatch = useDispatch();
+  let isMe = false;
+  let messageUserId = currentUserId;
+  const checkUserId = () => {
```

Comment:
This function usefull because when I sending data to server there is litle time period then message hasn`t userData from server but message already in redux and message jumped to left side. Current user message must be on right side and other user messages on left. This function solve this problem.

Reply:
Ok then, good code, but what for you wrap it in a function? It's used only once